errorHandling.java

```diff
@@ -0,0 +1,52 @@
+package sample;
+
+public class ErrorHandling {
+
+  
```

Comment:
// 1. Use Exceptions rather than returning code
  // Good 
  public User createUser(User user){
      try {
        loadUser(user.getId());
        throw new UserExistsException(user.getId())
      }
      catch (UserNotFoundException exception){
        // detail here
      }
      resolveGroup(user);
      return userRepository.save(user);
  }